.travis.yml

```diff
@@ -17,16 +17,17 @@ matrix:
     - php: 5.6
       env: DB=MYSQL CORE_RELEASE=3.5
     - php: 5.6
-      env: DB=MYSQL CORE_RELEASE=3.6
+      env: DB=MYSQL CORE_RELEASE=3.6 MAINTENANCE=^1
     - php: 7.0
-      env: DB=MYSQL CORE_RELEASE=3.6
+      env: DB=MYSQL CORE_RELEASE=3.6 PHPUNIT_COVERAGE_TEST=1
```

Comment:
We aren't running coverage in this branch at the moment, this might be a little misleading

Reply:
Yeah, it was more a mark in preparation for merging up to master. Can remove if you'd prefer.